### PR TITLE
Peter Lapicola

### PR DESCRIPTION
**Pull request by plapicola**
Pull request for independent challenge 'Museo'.

### CODE REVIEW

File: lib/curator.rb [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    end
+  end
+
+  def load_photographs(location)
+    file = File.new(location)
+    csv = CSV.new(file, headers: true,
+                        header_converters: :symbol,
+                        col_sep: ",,")
+    csv.read.each do |row|
+      add_photograph(row)
```
Comment (BrianZanti) [inline lib/curator.rb]: great job reusing this method.

File: lib/curator.rb [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    end
+  end
+
+  def add_artist(info)
+    artist = Artist.new(info)
+    @artists << artist
```
Comment (BrianZanti) [inline lib/curator.rb]: 👍

File: lib/curator.rb [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+  def find_photographs_by_artist(artist)
+    @photographs.find_all do |photo|
+      photo.artist_id == artist.id
+    end
+  end
+
+  def artists_with_multiple_photographs
+    @artists.find_all do |artist|
+      find_photographs_by_artist(artist).length > 1
```
Comment (BrianZanti) [inline lib/curator.rb]: Great job reusing this method 👍

File: test/artist_test.rb [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+    @artist = Artist.new(@attributes)
+  end
+
+  def test_it_exists
+    assert_instance_of Artist, @artist
+  end
+
+  def test_it_has_an_id
+    assert_equal @attributes[:id], @artist.id
```
Comment (BrianZanti) [inline test/artist_test.rb]: I'd prefer you be more explicit about the expected value. Just put "2"

File: test/curator_test.rb [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+  end
+
+  def test_it_can_add_artists
+    @curator.add_artist(@artist_1)
+    @curator.add_artist(@artist_2)
+
+    @curator.artists.each do |artist|
+      assert_instance_of Artist, artist
+    end
+    assert_equal 2, @curator.artists.count
```
Comment (BrianZanti) [inline test/curator_test.rb]: Looks good 👍

File: test/curator_test.rb [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+  end
+
+  def test_it_can_find_artists_by_id
+    @curator.add_artist(@artist_1)
+    @curator.add_artist(@artist_2)
+
+    assert_equal @artist_1[:name], @curator.find_artist_by_id("1").name
+    assert_nil @curator.find_artist_by_id("3")
```
Comment (BrianZanti) [inline test/curator_test.rb]: I like this edge case testing
